modules/ROOT/pages/upgrade-self-managed.adoc

```diff
@@ -0,0 +1,87 @@
+= Upgrade Runtime Fabric for Self-Managed Kubernetes
+
+To upgrade Anypoint Runtime Fabric for Self-Managed Kubernetes, you need to upgrade Runtime Fabric components and the `rtfctl` command-line utility.
+
+== About Component Upgrades
+
+Runtime Fabric component upgrades involve the internal services that run within the appliance used to facilitate communication to the Anypoint control plane, provide load balancing, and log and metrics forwarding.
+
+Runtime Fabric component upgrades are applied within Runtime Manager via a button that's visible when an upgrade is made available.
+
+Runtime Fabric component upgrades do not require adding a node.
+
+== About `rtfctl` Command-Line Utility Upgrades
+The `rtfctl` command-line utility is used to manage Runtime Fabric and perform appliance and node configuration upgrades.
+
+You upgrade the `rtfctl` utility from the command line.
+
+It is important to use the most current version of the `rtfctl` command-line utility.
+
+== Determine the Component Versions You Are Using
+
+From your local workstation (configured with a kubeconfig file) or a support bastion, run the following command:
+```
+.rtfctl version
+```
+
+Output includes:
+
+* The `rtfctl` command-line utility version in the `rtfctl` field
+* The Runtime Fabric component version in the `agent` field
+
+For example:
+```
+$ ./rtfctl version
+COMPONENT    VERSION
+rtfctl       0.3.122
+agent        1.9.6
+kubernetes   1.19.6-eks-49a6c0
+
+```
+
+[NOTE]
+The Runtime Fabric component version that is in use are also displayed in the *Runtime Fabrics* tab in Runtime Manager. If an upgrade is available, an "Upgrade to vx.x" message is displayed.
+
+== Apply the Upgrades
+
+To upgrade Anypoint Runtime Fabric for Self-Managed Kubernetes, you:
+
+. Upgrade Runtime Fabric Components
+. Upgrade the `rtfctl` command-line utility
```

Comment:
Gee, at this point I wonder if we are making this more complex than it needs to be, with all the small subtopics.
Seems like we could have...:

= Upgrade Runtime Fabric for Self-Managed Kubernetes
To upgrade Anypoint Runtime Fabric for Self-Managed Kubernetes, upgrade Runtime Fabric components and the `rtfctl` command-line utility.
Before upgrading, 
- Learn about the latest features
- Verify the current `rtfctl` and component `agent` versions.
- Move to a sandbox environment. (or incorporate this in to the steps, as the first one)

== Upgrade Runtime Fabric Components
Runtime Fabric components are.... 
To upgrade:
1. In a sandbox environment....
2.  
== Upgrade the `rtfctl` Command-Line Utility
The `rtfctl` command-line utility is....
To upgrade:
1. Do this
2. Do that

== Test in Sandbox and Move to Production???

Reply:
The remainder of my edits are based on sticking with the original composition/copyedits.

Reply:
Ok. I will come back to this in the future after you review the other topic (as I'm trying to get parts of them aligned for reuse purposes).